app/assets/javascripts/modules/collapsible.js

```diff
@@ -5,34 +5,46 @@
   function Collapsible(section){
     this.$section = section;
     this.$clickTarget = this.$section.find('.js-subsection-title');
-    this.$clickTarget.on('click', this.toggle.bind(this));
+    this.$clickTarget.on('click', this.handleClick.bind(this));
     this.addToggle();
 
     this.$section.on('focus', '.js-subsection-body a', this.showSectionWhenLinkFocused.bind(this));
   }
 
+  Collapsible.prototype.handleClick = function handleClick(){
+    this.toggle()
+    this.toggleAriaExpanded()
+  };
+
+  Collapsible.prototype.toggleAriaExpanded = function toggleAriaExpanded() {
+    // $click target is the entire h2 so we just need to get the button
+    var $button = $(this.$clickTarget[0].querySelector('button'))
+    $button.attr('aria-expanded', ($button.attr('aria-expanded') === "false" ? true : false));
+  };
+
   Collapsible.prototype.showSectionWhenLinkFocused = function() {
     if (this.$section.is('.closed')) {
       this.$section.toggleClass('closed');
     }
   };
 
   Collapsible.prototype.addToggle = function addToggle(){
-    var $toggleHTML = $("<span class='js-toggle'></span>");
+    var $toggleHTML = $("<span class='js-toggle' aria-hidden='true'></span>");
     this.$clickTarget.append($toggleHTML);
   };
 
-  Collapsible.prototype.toggle = function toggle(event){
+  Collapsible.prototype.toggle = function toggle(){
     this.$section.toggleClass('closed');
-    event.preventDefault();
   };
 
   Collapsible.prototype.close = function close(){
     this.$section.addClass('closed');
+    this.$clickTarget[0].querySelector('button').setAttribute('aria-expanded', 'false');
   };
 
   Collapsible.prototype.open = function open(){
     this.$section.removeClass('closed');
+    this.$clickTarget[0].querySelector('button').setAttribute('aria-expanded', 'true');
```

Comment:
This can be cached in the constructor now I think? Would make the rest of the code much easier to read 👍 

```js
this.$button = this.$clickTarget[0].querySelector('button');
```

Reply:
done